airflow/config_templates/default_airflow.cfg

```diff
@@ -1077,6 +1077,10 @@ standalone_dag_processor = False
 # in database. Contains maximum number of callbacks that are fetched during a single loop.
 max_callbacks_per_loop = 20
 
+# Only applicable if `[scheduler]standalone_dag_processor` is true.
+# Time in seconds after which dags, which were not updated by Dag Processor are deactivated.
+dag_stale_not_seen_duration = 600
```

Comment:
`dag_stale_not_seen_duration` --> any suggestion for a better name? this config name isn't easy understand

Reply:
eh.. naming things...

Let me be wild on that one: 

`deactivation_time_for_missing_dags_in_standalone_dag_processor_mode`

Reply:
Another suggestion from @ashb was `mark_dag_stale_not_seen_in` 
Is it better?

Reply:
All of them are awful :)

Reply:
actually yours @potiuk helped me to understand the meaning of that parameter ;p

Reply:
Horrible names can also be best :)